quizzes/midterm.md

```diff
@@ -68,20 +78,25 @@
 
 
 1. Demonstrate using multiple logical operators (||, &&)
-
+(true || true && false) true
+(false && false || true) false
 
 1. What is the result of calling sort?
 
 	```ruby
 	["one fish", "two fish", "red fish", "blue fish"].sort
 	```
-
+	"blue fish", "one fish", "red fish", "two fish"
 
 1. Write a hash that has the following keys :name, :super_power, :weakness
-
+hash.new{
+name: "joe",
+super_power: "mind control",
+weakness: "hunger"
+}
 
 1. T/F You can set a default value for a Hash
-
+true
 
 
 1. What is the resulting hash?
```

Comment:
4/10 looking for the resulting hash. You get to the answer and then contradict yourself.